subsys/net/l2/wifi/wifi_shell.c

```diff
@@ -2909,6 +2978,10 @@ SHELL_STATIC_SUBCMD_SET_CREATE(wifi_commands,
 		cmd_wifi_btm_query,
 		2, 0),
 #endif
+	SHELL_CMD_ARG(11k_enable, NULL, "<0/1>\n",
```

Comment:
I am not a big fan of just giving 0/1 here, it would be more logical if user could just say "enable" or "disable".
So `wifi 11k enable` or `wifi 11k disable` or similar.

Reply:
Updated. The new command "wifi 11k [enable/disable]" can be used to enable or disable 11k, instead of giving paramter 0/1.